R/diagnosis_helper_functions.R

```diff
@@ -106,57 +106,52 @@ reshape_diagnosis <- function(diagnosis, digits = 2, select = NULL) {
   
   diagnosands_df <- diagnosis$diagnosands
   
-  # If no bootstrapping
-  if (is.null(diagnosis$bootstrap_replicates)) {
-    # Make names nicer
-    names(diagnosands_df) <-
-      gsub("\\b(se[(]|sd |rmse|[[:alpha:]])",
-           "\\U\\1",
-           gsub("_", " ", names(diagnosands_df)),
-           perl = TRUE)
-    rownames(diagnosands_df) <- NULL
+  if(!is.null(diagnosis$bootstrap_replicates)){
```

Comment:
is.data.frame

Reply:
This is a very large if/else block - it was easier to read when the small edge case was close to the if statement, instead of 50 lines away. Probably should refactor this large block to be clean_bootstrap_replicates or similar - use rstudio's extract function command.

Reply:
ok thanks will do